docs/src/model_setup/boundary_conditions.md

```diff
@@ -293,7 +293,7 @@ julia> u_bcs = FieldBoundaryConditions(top = ValueBoundaryCondition(+0.1),
 julia> T_bcs = FieldBoundaryConditions(top = ValueBoundaryCondition(20),
                                        bottom = GradientBoundaryCondition(0.01));
 
-julia> model = NonhydrostaticModel(grid=grid, boundary_conditions=(u=u_bcs, T=T_bcs))
+julia> model = NonhydrostaticModel(grid=grid, boundary_conditions=(u=u_bcs, T=T_bcs), buoyancy=SeawaterBuoyancy(), tracers=(:T, :S))
```

Comment:
why does this example need to prescribe salinity?
why not

```suggestion
julia> model = NonhydrostaticModel(grid=grid, tracers=(:T, ), boundary_conditions=(u=u_bcs, T=T_bcs))
```

Reply:
I thought it would be weird to have `:T` (which we always use to denote temperature, an active tracer) and no buyancy model, effectively implying that temperature is a passive tracer. I agree this is a minor issue, since we don't really anything with `T` here besides imposing BC.

So in this particular case I'd advocate for either leaving this as it is, or maybe changing `T` to a passive tracer `C`?

Reply:
Why not `c` as a passive tracer?
Or does the demo here requires an active tracer?

Reply:
Change to `c` (not `C`) for tracer `c`oncentration.

Reply:
That's what I was suggesting. Why `c` and not `C` though? Is `C` reserved for something?